### ISSUE TITLE
Clarify MinIO development settings

### ISSUE THREAD

**Issue by astrojuanlu** [readthedocs/readthedocs.org#8090]
In https://github.com/readthedocs/readthedocs.org/pull/8082#issuecomment-816832362, @cocobennett followed the steps to configure MinIO for local development and still ended up with an incomplete environment. In fact, I had made the same mistake two weeks early when I just joined the company. @stsewd [clarified later](https://github.com/readthedocs/readthedocs.org/pull/8082#issuecomment-816875657), but this should be present in the docs.

In particular, I think we should replace this:

https://github.com/readthedocs/readthedocs.org/blame/eb682d8b8f480e96e7478d968aa12e52f4118063/docs/development/install.rst#L66-L71

by something that doesn't assume that the user knows how MinIO works.

**Comment by astrojuanlu** [comment]
Also, instead of waiting for the first build to create the `media` bucket, we can either create it manually or automate it on init somehow. What do you think @stsewd @humitos?

**Comment by humitos** [comment]
> instead of waiting for the first build to create the `media` bucket, we can either create it manually or automate it on init somehow

Yes. This is something we want to do. We used to do this when using Azurite (Azure Blob Storage emulator) but when we migrated to S3 we removed this because the bucket were automatically created on the first usage. However, that won't be true anymore when we upgrade django-storages. See https://github.com/readthedocs/common/pull/80#issuecomment-759281298

Summarizing, we want that feature back :smile:
